selenium/automate_chrome.sh

```diff
@@ -3,28 +3,54 @@ set -e
 input=$1
 driver_version=$2
 tag=$3
+channel=${4:-"stable"}
 
 if [ -z "$1" -o -z "$2" -o -z "$3" ]; then
-    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version>'
+    echo 'Usage: ./automate_chrome.sh <browser_version|package_file> <chromedriver_version> <tag_version> [{stable|dev|beta}]'
     exit 1
 fi
 set -x
 
 browser_version=$input
 method="chrome/apt"
 if [ -f "$input" ]; then
-    cp "$input" chrome/local/google-chrome-stable.deb
+    cp "$input" chrome/local/google-chrome.deb
     filename=$(echo "$input" | awk -F '/' '{print $NF}')
     browser_version=$(echo $filename | awk -F '_' '{print $2}' | awk -F '-' '{print $1}')
     method="chrome/local"
 fi
+if [ "$browser_version" == "latest" ]; then
```

Comment:
Instead of `latest`, please use `stable`?

Reply:
Refactored